lib/matplotlib/backends/backend_webagg_core.py

```diff
@@ -403,7 +403,8 @@ def get_javascript(cls, stream=None):
         for filetype, ext in sorted(FigureCanvasWebAggCore.
                                     get_supported_filetypes_grouped().
                                     items()):
-            extensions.append(ext[0])
+            if not ext[0] == 'pgf':  # pgf does not support BytesIO
```

Comment:
If possible, I think it's preferable to fix this at the source (having `pgf` support BytesIO objects, if that's possible), rather than papering around it here.